docs/data-sources/dns_records.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud_dns_records Data Source - terraform-provider-anxcloud"
+subcategory: ""
+description: |-
+  Provides DNS records for a specified zone.
+---
+
+# anxcloud_dns_records (Data Source)
+
+Provides DNS records for a specified zone.
+
+## Example Usage
+
+```terraform
+data "anxcloud_dns_records" "example" {
+  dns_zone = "example.com"
```

Comment:
the schema below shows `zone_name` instead of `dns_zone` if I read it correctly?

Reply:
Good catch!